src/Cli.Tests/ModuleInitializer.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+using Azure.DataApiBuilder.Config.ObjectModel;
+
+namespace Cli.Tests;
+
+/// <summary>
+/// Setup global settings for the test project.
+/// </summary>
+static class ModuleInitializer
+{
+    /// <summary>
+    /// Initialize the Verifier settings we used for the project, such as what fields to ignore
+    /// when comparing objects and how we will name the snapshot files.
+    /// </summary>
+    [ModuleInitializer]
+    public static void Init()
+    {
+        VerifierSettings.IgnoreMember<DataSource>(dataSource => dataSource.ConnectionString);
```

Comment:
```suggestion
        // Ignore specific members of the DataSource and RuntimeConfig classes
        VerifierSettings.IgnoreMember<DataSource>(dataSource => dataSource.ConnectionString);
```

Reply:
Updated this file with better explanation of what each step does.